study.md

```diff
@@ -21,18 +21,52 @@ There is no requirement to follow any links from these pages.
 After doing all the required readings please tell us anything you were confused about.
 
 ```md
-<!-- your answer here -->
+Interrupting a function by return; - when the code get executed to the "return;" what will trigger it to go back to print out "B" ? Also I would think that the result will not get any numbers with B then when return; gets in.
+
+function counter() {
+  for (var count = 1; ; count++) {  // infinite loop
+    console.log(count + "A"); // until 5
+      if (count === 5) {
+        return;
+      }
+      console.log(count + "B");  // until 4
+    }
+  console.log(count + "C");  // never appears
+}
+
+counter();
+
+// Output:
+// 1A
+// 1B
+// 2A
+// 2B
+// 3A
+// 3B
+// 4A
+// 4B
+// 5A
+
```

Comment:
The numbers with Bs following them print because ```console.log(count + "B");``` always executes while count is not equal to 5.  Once ```count``` is equal to 5, the if-block's condition is true and executes the ```return;``` statement, which returns control back to the global scope.